sdf/final-project/technical.md

```diff
@@ -6,102 +6,138 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
 - Project Name:
 - Reviewer Name:
 
-## Documentation
-- [ ] **Clarity and Completeness**: Is the README clear, well-written, and comprehensive? Does it effectively explain the project's purpose, features, and limitations?
-- [ ] **Setup and Installation Instructions**: Does the documentation include detailed setup and installation instructions, ensuring a new developer can get the project running locally without external help?
-- [ ] **Contribution Guidelines**: Are there clear contribution guidelines? Do they outline how developers can contribute to the project, including coding conventions, branch naming conventions, and the pull request process?
-- [ ] **Markdown**: Is the README formatted using Markdown?
-- [ ] **Naming**: Is the repository name relevant to the project?
+## Technical Score (/80):
+- Readme (/10):
+- Version Control (/10):
+- Code Hygiene (/7):
+- Patterns of Enterprise Applications (/8):
+- Frontend (/10):
+- Backend (/10):
+- Security and Authorization (/5):
+- Ambitious Features (/10):
+- Enrichment (/10):
+
+## Additional overall comments for the entire review may be added below:
 
-### Bonus (max 2pts)
-- [ ] **Visual Aids**: Does the documentation include visual aids such as screenshots, diagrams (e.g., architecture diagrams, ERDs), or GIFs to clarify complex concepts or demonstrate features?
-- [ ] **FAQs or Troubleshooting**: Is there an FAQs or Troubleshooting section that addresses common issues, questions, or obstacles users or new contributors might face?
-- [ ] **API Documentation (if applicable)**: For projects with an API, is there clear and detailed API documentation, including endpoints, request/response formats, and authentication methods?
 
 
-### Score (/5):
+
+
+## Readme (max: 10 points)
+- [ ] **Markdown**: Is the README formatted using Markdown?
+- [ ] **Naming**: Is the repository name relevant to the project?
+- [ ] **1-liner**: Is there a 1-liner briefly describing the project?
+- [ ] **Instructions**: Are there detailed setup and installation instructions, ensuring a new developer can get the project running locally without external help?
+- [ ] **Configuration**: Are configuration instructions provided, such as environment variables or configuration files that need to be set up?
+- [ ] **Contribution**: Are there clear contribution guidelines? Do they outline how developers can contribute to the project, including coding conventions, branch naming conventions, and the pull request process?
+- [ ] **ERD**: Does the documentation include visual an ERD?
+- [ ] **Troubleshooting**: Is there an FAQs or Troubleshooting section that addresses common issues, questions, or obstacles users or new contributors might face?
+- [ ] **Screenshots**: Are there screenshots of the application in the README?
```

Comment:
maybe change to "visual aids"? eg Are there visual aids (diagrams, screenshots, etc.) that would help developers quickly ramp on to the project?

Reply:
Done